### DISCUSSION TITLE
A New Maintainer For yadm

### DISCUSSION THREAD

**Discussion by TheLocehiliosan** [TheLocehiliosan/yadm#500] [Announcements]
After thoughtful consideration, I am stepping down as the primary maintainer of yadm, and I am pleased to announce that Erik Flodin (@erijo) will be taking on this responsibility.

Erik has been a valuable contributor to yadm, and his past work has already made a lasting impact on the project. I have great admiration for his contributions—both in technical skill and his dedication to improving the tool. I’m truly grateful that he is willing to step into this responsibility, and it’s reassuring to know that the project will be in such capable hands.

This decision comes as I find myself unable to devote the time and energy that yadm deserves. Changes in my personal and professional life have limited my availability, making it difficult to keep up with the demands of maintaining an open-source project. While I’ll be taking a step back, I’ll continue to support Erik as he takes on this role, and I’ll remain involved, contributing in ways that align with my current capacity.

This change marks an exciting new chapter for yadm, and I’m confident that Erik’s leadership will bring fresh energy and perspective to the project.

Thank you to everyone who has supported yadm and contributed to making it what it is today. I deeply appreciate the community that has grown around this project, and I look forward to seeing yadm continue to grow and evolve.

-Tim
@TheLocehiliosan

**Comment by acid-bong** [comment]
Congrats to Erik and a biiiiig-ass thank you, Tim, for what you've done. YADM is truly the best dotfile manager out there

**Comment by rasa** [comment]
Thanks @TheLocehiliosan, for all your tireless efforts you put into yadm. It's always been a joy to use.

**Comment by erijo** [comment]
Yes, a big thank you Tim for what you've done with yadm and for entrusting it to me! I'll do my best to keep it rocking!
